implementations/rust/ockam/ockam_command/src/project/enroll.rs

```diff
@@ -166,7 +173,10 @@ impl Runner {
                 .with_timeout(Duration::from_secs(ORCHESTRATOR_RESTART_TIMEOUT)),
             );
             let token = client.create_token(self.cmd.attributes()?).await?;
-            println!("{}", token.to_string())
+
+            let ticket = EnrollmentTicket::new(token, project, trust_context);
+            let ticket_serialized = hex::encode(serde_json::to_vec(&ticket)?);
+            print!("{}", ticket_serialized)
```

Comment:
Not taking any stance about opt-in vs opt-out, but should there still be an available version to get human-readable output?